packages/config/src/config.js

```diff
@@ -63,10 +63,15 @@ module.exports = ({
                         outputPath: 'fonts/'
                     }
                 }]
+            },
+            {
+                test: /\.mjs$/,
+                include: /node_modules/,
+                type: 'javascript/auto'
             }]
         },
         resolve: {
-            extensions: [ '.ts', '.tsx', '.js', '.scss' ]
+            extensions: [ '.ts', '.tsx', '.mjs', '.js', '.scss' ]
```

Comment:
Aren't ECMAScript modules an experimental feature?

Reply:
That's actually what the preferes solution is about https://github.com/graphql/graphql-js/issues/1272

The problem is that you can't import named exports multiple times and this was breaking it.